script/upgrades/MU01/deploy/MU01-05-Deploy-BreakerBox.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.5.13;
+
+import { Script } from "script/utils/Script.sol";
+import { Chain } from "script/utils/Chain.sol";
+import { console2 } from "forge-std/Script.sol";
+
+import { BreakerBox } from "mento-core/contracts/BreakerBox.sol";
+import { ISortedOracles } from "mento-core/contracts/interfaces/ISortedOracles.sol";
+
+/*
+ forge script MU01_DeployBreakerBox --rpc-url $RPC_URL
+                             --broadcast --legacy 
+                             --verify --verifier sourcify 
+*/
+contract MU01_DeployBreakerBox is Script {
```

Comment:
This is no longer MU01, so imho we should rename this contract and this file to something else. 

How about `MU02-00-DeployBreakerBox` and `MU02_DeployBreakerBox`?